cucumber/authenticators_jwt/features/authn_jwt.feature

```diff
@@ -130,3 +130,58 @@ Feature: JWT Authenticator - JWKs Basic sanity
     """
     CONJ00100E Annotation, 'custom-claim', is empty
     """
+
+  Scenario: Ignore invalid annotations
+    Given I have a "variable" resource called "test-variable"
+    Given I extend the policy with:
+    """
+    - !host
+      id: myapp
+      annotations:
+        authn-jwt/raw: invalid
+        authn-jwt: invalid
+        authn-jwt/raw/sub/sub: invalid
+        authn-jwt/raw2/sub: invalid
+        authn-jwt/raw/sub: valid
+        authn-jwt/raw/namespace-id: valid
+        authn-jwt/raw/project-path: valid
+    """
+    And I permit host "myapp" to "execute" it
+    And I add the secret value "test-secret" to the resource "cucumber:variable:test-variable"
+    And I issue a JWT token:
+    """
+    {
+      "user":"host/myapp",
+      "project-id": "myproject",
+      "sub": "valid",
+      "namespace-id": "valid",
+      "project-path": "valid"
+    }
+    """
+    And I save my place in the log file
+    When I authenticate via authn-jwt with the JWT token
+    Then the HTTP response status code is 200
```

Comment:
Before verifying logs try to verify you retrieve the secret well

Reply:
I dont understand what you mean, which secrets do i need to verify? there are other tests to check the valid flow

Reply:
you update `cucumber:variable:test-variable` and you want to verify you can get this value , otherwise why you update it ?

Reply:
youre right, i need to delete `I add the secret value "test-secret" to the resource "cucumber:variable:test-variable"` because i has nothing to do with my text, i just copied it from another test to simulate a working authentication

Reply:
Why not verify you get the secret, it fast and cost nothing and will get more confidence here.